client/packages/programs/src/JsonForms/components/Search/Search.tsx

```diff
@@ -21,18 +22,18 @@ const UserOptions = z.object({
    * Pattern for formatting options list items (e.g. "${firstName} ${lastName}")
    */
   optionString: z.string().optional(),
-  /**
-   * Pattern for formatting selected result (as above)
-   */
-  displayString: z.string().optional(),
   /**
    * List of fields to save in document data (from selected item object)
    */
   saveFields: z.array(z.string()).optional(),
   /**
-   * Text to show in input field before user entry
+   * Child form elements
    */
-  placeholderText: z.string().optional(),
+  elements: z.array(z.any()),
```

Comment:
Is "any" okay here? Seems like it would be a lot of work for little gain to try and define all the possible "UISchema" structures using Zod functionality. (Especially since the existing "UISchemaElement" type is incomplete)

Reply:
Agree, have done the same elsewhere

Reply:
Nit: could you please add a doc comment for UserOptions with an high level overview what it does?

Reply:
db411c920